WIP-CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 - The NetworkPolicy Dashboard have been updated to be more clear
 - Upgraded the Starboard-operator helm chart to `0.10.11` which upgrades the app version to `0.15.11`
+- Upgraded the cert-manager helm chart to `v1.11.0` which upgrades the app version to `v1.11.0`
```

Comment:
where there any braking changes or other changes that could affect us or the customers?

Reply:
> where there any braking changes or other changes that could affect us or the customers?

Updated the changelog with the breaking changes I could find from the release notes